notebooks/notebook.ipynb

```diff
@@ -1,5 +1,12 @@
 {
```

Comment:
<p>Could you please explain why the plots on the published paper look different?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/repro-challenge-team-3/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>Is this fix on the github version yet? I just did a git pull and didn't seem to get any updates. It would be good to be able to test running with this locally please</p>

Reply:
@RachelFurner authors have incorporated changes into the `main` branch, so please explore the notebook locally from this branch instead of the `review` branch.

Reply:
closing conversation, resolved